qiskit-qir/setup.cfg

```diff
@@ -0,0 +1,34 @@
+[metadata]
+name = qiskit-qir
+version = 0.0.1
```

Comment:
Will this package share common version numbers with any other components, or is it standalone?